og_migrate/migrations/d7_og_group.yml

```diff
@@ -3,7 +3,7 @@ label: Og groups
 migration_tags:
   - Drupal 7
 source:
-  plugin: d6_og_group
+  plugin: d7_og_group
```

Comment:
Nice catch.

This also caused the OgFieldD7MigrateTest to fail because of missing migration dependencies. So we need to update that as well.

I probably removed them at some point.

Reply:
I can't get the test to work.
I keep getting the error "Missing file with ID 1" when I activate the d7_node migration with executeMigrations, but can't figure out where the file is being used.
(I'm new to D8 and have a bit of trouble finding my way around at the moment).
Will try to figure this out by the end of the week.

Reply:
Hmm, I haven't encountered that before when running tests. 

It can take a bit to get a test environment setup and sometimes a virtual machine or docker environment might need a bit of extra configuration to run the test suite (like /etc/hosts changes).

Reply:
I am using phpunit in VSCode, and the setup seems to work fine as other tests run without a problem.
The issue seems to be with the d7_node migration dependency, which seems to require a file that is apparently not created.
So I guess I'm missing a module or a migration in the test.
Below is the content of OgFieldD7MigrateTest.php so far, which keeps failing with the above mentioned error when it gets to the d7_node migration in setUp.

```php
<?php

namespace Drupal\Tests\og\Kernel\Migrate;

use Drupal\field\Entity\FieldConfig;
use Drupal\og\Entity\OgMembership;
use Drupal\taxonomy\Entity\Vocabulary;
use Drupal\Tests\migrate\Kernel\NodeCommentCombinationTrait;
use Drupal\Tests\migrate_drupal\Kernel\d7\MigrateDrupal7TestBase;

/**
 * Tests organic group field migration.
 *
 * @group og_migrate
 */
class OgFieldD7MigrateTest extends MigrateDrupal7TestBase {

  use NodeCommentCombinationTrait;

  /**
   * {@inheritdoc}
   */
  public static $modules = [
    'text',
    'entity_reference',
    'telephone',
    'datetime',
    'file',
    'image',
    'link',
    'filter',
    'menu_ui',
    'comment',
    'node',
    'system',
    'taxonomy',
    'og',
    'og_ui',
    'og_migrate',
  ];

  /**
   * {@inheritdoc}
   */
  protected function setUp() {
    parent::setUp();

    $this->loadFixture(__DIR__ . '/../../../fixtures/drupal7.php');
    $this->installConfig(static::$modules);
    $this->installEntitySchema('og_membership_type');
    $this->installEntitySchema('og_membership');
    $this->installEntitySchema('og_role');
    $this->installEntitySchema('node');
    $this->installEntitySchema('file');
    $this->installEntitySchema('taxonomy_term');
    $this->installEntitySchema('taxonomy_vocabulary');
    $this->createNodeCommentCombination('page');
    $this->createNodeCommentCombination('article');
    $this->createNodeCommentCombination('blog');
    $this->createNodeCommentCombination('book');
    $this->createNodeCommentCombination('forum', 'comment_forum');
    $this->createNodeCommentCombination('test_content_type');
    Vocabulary::create(['vid' => 'test_vocabulary'])->save();
    $this->executeMigrations([
      'd7_field',
      'd7_field_instance',
      'd7_user_role',
      'd7_user',
      'd7_node_type',
      'd7_node',
      'd7_og_group',
      'd7_og_field_instance',
    ]);
  }

  /**
   * Asserts that the og-related fields were migrated properly.
   */
  public function testOgAudienceFields() {
    $nodeField = FieldConfig::load('node.article.og_audience');
    $fieldDefinitions = $this->container->get('entity_field.manager')->getFieldDefinitions('node', 'article');

    $this->assertEquals('og_standard_reference', $nodeField->getType());
    $this->assertArrayHasKey('og_audience', $fieldDefinitions);

    $forumField = FieldConfig::load('node.forum.og_audience');
    $fieldDefinitions = $this->container->get('entity_field.manager')->getFieldDefinitions('node', 'forum');

    $this->assertEquals('og_standard_reference', $forumField->getType());
    $this->assertArrayHasKey('og_audience', $fieldDefinitions);
  }

}

```

Reply:
Finally figured it out: we need to use the FileMigrationSetupTrait in the test for files to be imported.

Reply:
https://github.com/mradcliffe/og/pull/1/commits/e33f3ccfc774793e1071007be5fc8a0d1039e98b fixes the OgFieldD7MigrateTest test

Reply:
I seem to have ran the wrong test by accident yesterday.
The OgFieldD7MigrateTest still fails, but I'm working on it...